_posts/2017-02-09-rapid-mysql-imports-docker.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "Dramatically improve MySQL database import time on Docker stacks"
```

Comment:
Just making a note that we're standardizing to capitalize title words excluding articles, prepositions, etc!